web/pgadmin/static/js/components/FormComponents.jsx

```diff
@@ -1115,15 +1110,17 @@ PlainString.propTypes = {
 };
 
 export function FormNote({ text, className, controlProps }) {
-  const classes = useStyles();
+
   /* If raw, then remove the styles and icon */
   return (
-    <Box className={className}>
-      <Paper elevation={0} className={controlProps?.raw ? '' : classes.noteRoot}>
-        {!controlProps?.raw && <Box paddingRight="0.25rem"><DescriptionIcon fontSize="small" /></Box>}
-        <Box>{HTMLReactParse(text || '')}</Box>
-      </Paper>
-    </Box>
+    <Root>
+      <Box className={className}>
```

Comment:
Can't we directly style the Box here?

Reply:
Need to add another styling element then.